sites/partners/src/listings/PaperListingForm/sections/BuildingDetails.tsx

```diff
@@ -147,7 +152,12 @@ const BuildingDetails = ({
             label={t("application.contact.streetAddress")}
             name={"buildingAddress.street"}
             id={"buildingAddress.street"}
+            error={errors?.buildingAddress?.street !== undefined}
+            errorMessage={errors?.buildingAddress?.street?.message}
```

Comment:
Wondering if it would be useful to have a function to handle these that would accept the path like "buildingAddress.street", so we don't have to do the same checks everywhere. Something like `hasError: (path: string) => boolean` and `errorMessage: (path: string) => string`